manifest/ocischema/index.go

```diff
@@ -0,0 +1,174 @@
+package ocischema
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/distribution/distribution/v3"
+	"github.com/distribution/distribution/v3/manifest"
+	"github.com/opencontainers/go-digest"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// OCISchemaVersion provides a pre-initialized version structure for this
+// packages OCIschema version of the manifest.
+var OCISchemaVersion = manifest.Versioned{
+	SchemaVersion: 2,
```

Comment:
Same comment as earlier. Feel free to ignore me, but I figured it might be worth changing it as we use this _const_ value in a few places in this file.

Reply:
I ended up using this definition as the source of truth for the other uses in the Index. I know it's not const because it's a struct, I can isolate it further to a read const if you like?

Reply:
I think this is fine.